templates/kubernetes/terraform/environments/prod/application_iam_policy.tf

```diff
@@ -8,6 +8,7 @@ data "aws_iam_policy_document" "resource_access_backendservice" {
     ]
     resources = ["arn:aws:ec2:::prod-*"]
   }
+<% if eq (index .Params `fileUploads`) "yes" %>
```

Comment:
Realized it wasn't useful to leave this here if the user chose no for this option.